pkg/maps/ctmap/ctmap.go

```diff
@@ -349,22 +352,33 @@ func statStartGc(m *bpf.Map, family gcFamily) gcStats {
 
 func (s *gcStats) finish() {
 	s.finished = time.Now()
-
+	duration := s.finished.Sub(s.started)
+	family := s.family.String()
 	switch s.family {
 	case gcFamilyIPv6:
 		metrics.DatapathErrors.With(labelIPv6CTDumpInterrupts).Add(float64(s.interrupted))
 	case gcFamilyIPv4:
 		metrics.DatapathErrors.With(labelIPv4CTDumpInterrupts).Add(float64(s.interrupted))
 	}
 
-	if !s.completed {
+	var status string
+	if s.completed {
+		status = "completed"
+		metrics.ConntrackGCSize.WithLabelValues(family, metricsAlive).Set(float64(s.aliveEntries))
+		metrics.ConntrackGCSize.WithLabelValues(family, metricsDeleted).Set(float64(s.deleted))
+	} else {
+		status = "uncompleted"
```

Comment:
I think we may get statistics even in the incomplete case here, maybe we should also increment them in this other path?

Reply:
The caveat would be that if the dump is incomplete, maybe the statistics aren't entirely correct (though reporting what's known could still be useful).

Reply:
No, a previous comment said that we shouldn't report if it's not successful. The  alive can be 0 if it is not completed, so I understand that we don't want to report that 0 entries in the CT. 

The other thing is to use s.Count, the number when start, but a previous comment also said that we should use alive. 

This is the best way that I found, sorry :-(